package-lock.json

```diff
@@ -22,25 +22,25 @@
       }
     },
     "@babel/compat-data": {
-      "version": "7.18.6",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.18.6.tgz",
-      "integrity": "sha512-tzulrgDT0QD6U7BJ4TKVk2SDDg7wlP39P9yAx1RfLy7vP/7rsDRlWVfbWxElslu56+r7QOhB2NSDsabYYruoZQ=="
```

Comment:
Package-lock changes shouldn't be committed to PRs, under most circumstances--especially since it looks like this is rolling things back to earlier versions. If there's not a need for this, check out the current state of these files from current develop to roll back these changes.